src/sentry/static/sentry/app/views/performance/transactionVitals/vitalCard.tsx

```diff
@@ -414,29 +425,67 @@ class VitalCard extends React.Component<Props, State> {
       this.state.refPixelRect!
     );
     if (failurePixel === null) {
-      return;
+      return null;
     }
 
-    series.markArea = MarkArea({
-      animationDuration: 200,
+    const topRightPixel = mapPoint(
+      {
+        // subtract 0.5 to get on the right side of the right most bar
+        x: chartData.length - 0.5,
+        y: Math.max(...chartData.map(data => data.count)) || 1,
+      },
+      this.state.refDataRect!,
+      this.state.refPixelRect!
+    );
+    if (topRightPixel === null) {
+      return null;
+    }
+
+    // Using a MarkArea means that hovering over the interior of the area
+    // will trigger the tooltip for the MarkArea, making it impossible to
+    // see outliers with the tooltip. So we get around this using lines.
+    const markLine = MarkLine({
+      animation: false,
       data: [
+        // left
+        [
+          {x: failurePixel.x, y: failurePixel.y},
+          {x: failurePixel.x, y: topRightPixel.y},
+        ] as any,
+        // top
+        [
+          {x: failurePixel.x, y: topRightPixel.y},
+          {x: topRightPixel.x, y: topRightPixel.y},
+        ] as any,
+        // right
         [
-          {x: failurePixel.x, yAxis: 0},
-          {x: 'max', y: 'max'},
+          {x: topRightPixel.x, y: topRightPixel.y},
+          {x: topRightPixel.x, y: failurePixel.y},
+        ] as any,
+        // bottom
```

Comment:
Do we need a bottom? Having a bottom line partly obscures the outlier values, I think the left/right/top outline the box well enough.

Reply:
I'll remove the bottom line 👍